static/js/tools/scatter/context.ts

```diff
@@ -164,8 +166,9 @@ const FieldToAbbreviation = {
   lowerBound: "lb",
   upperBound: "ub",
 
-  // Quadrant fields
+  // DisplayOptions fields
   isQuadrant: "qd",
```

Comment:
nit: maybe these can also be showQuandrant and showLabels

Reply:
done!